python/lsst/ts/MTAirCompressor/aircompressor_csc.py

```diff
@@ -118,30 +52,45 @@ class MTAirCompressorCsc(salobj.BaseCsc):
     valid_simulation_modes: typing.Sequence[int] = (0, 1)
 
     def __init__(
-        self, index: int, initial_state=salobj.State.DISABLED, simulation_mode: int = 0
+        self,
+        index: int,
+        initial_state=salobj.State.DISABLED,
+        simulation_mode: int = 0,
     ):
         super().__init__(
             name="MTAirCompressor",
             index=index,
-            simulation_mode=simulation_mode,
             initial_state=initial_state,
+            simulation_mode=simulation_mode,
         )
 
+        self.grace_period = 600  # TODO should be configurable - DM-35280
+
         self.first_run = True
-        self.client = None
+        self.model = None
         self.simulator = None
         self.simulator_future = None
-        self.telemetry_task = utils.make_done_future()
+        self._start_by_remote = False
```

Comment:
Regarding `simulator_future` above: I strongly suggest making it a done Future so you don't have to test it for None and can always cancel it. The utils package has `make_done_future` for this purpose, though it's also a 2-liner to do it manually (create the Future and then set its result).